### PR TITLE
nixos wifi router blog post

### PR DESCRIPTION
**Pull request by scopatz**

### CODE REVIEW

File: posts/2020/06/nixos-rpi-wifi-router.md [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+In many cases, the advantages here outweigh the disadvantages. It is without
+question that [Docker](https://www.docker.com/), [CoreOS](http://coreos.com/),
+and [conda](https://docs.conda.io/en/latest/) all owe a lot of conceptual
+inspiration to the work Nix has been performing for years.
+
+Of course, even a functional OS has to have an escape valve. This is called
+`nix-env` and is a command line utility for creating & managing environments
+(a collection of packages) in an existing OS.
+
+:warning: **Do not use `nix-env`!** :warning:
```
Comment (rgommers) [inline posts/2020/06/nixos-rpi-wifi-router.md]: this required a plugin, added

File: posts/2020/06/nixos-rpi-wifi-router.md [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+$ mkdir -p ~/mount
+$ sudo mount /dev/sdb2 ~/mount
+$ sudo mkdir -p ~/mount/etc/nixos
+$ sudo cp ~/Downloads/config.nix ~/mount/etc/nixos/configuration.nix
+$ sudo umount ~/mount
+```
+
+**Fifth**, now unplug the SD card from your main machine, plug it into the
+Raspberry Pi! Attach the ethernet, keyboard, monitor, and power supply to the
+Pi you will be booting up into your first NixOS! :tada:
```
Comment (rgommers) [inline posts/2020/06/nixos-rpi-wifi-router.md]: This didn't turn into an emoji, plus lower down `:copyright:`, `:wave:` have the same problem. Some unicode symbols are recognized by Nikola (copyright, registered), the image ones are best handled by going to Emojipedia (e.g. https://emojipedia.org/waving-hand/), using the "copy" button there, and pasting into VSCode.

**Comment by rgommers** [comment]
I rebased, updated the data, fixed some textual issues and merged. Plus fixed URL to reflects it's July in gh-87. Deployed at https://labs.quansight.org/blog/2020/07/nixos-rpi-wifi-router/index.html (somehow the redirect on https://labs.quansight.org/blog/2020/07/nixos-rpi-wifi-router/ broke again, like 2 weeks ago - grrr Nikola is starting to annoy me).

**Comment by rgommers** [comment]
Alright, that's a Cloudflare thing I think. It works on my phone over 4G but not on a second machine on the same wifi. Moving posts to a different URL is a little tricky. Anyway, should work for others.

**Comment by scopatz** [comment]
Thanks for the cleanup @rgommers!

**Pull request merged**